flang/test/Transforms/debug-complex-2.f90

```diff
@@ -0,0 +1,28 @@
+! RUN: %flang_fc1 -emit-fir -debug-info-kind=standalone -mmlir --mlir-print-debuginfo %s -o - | \
```

Comment:
Does this test anything different from `debug-complex-1.fir`?

Reply:
I was going to remove it after the discussion in #91582. Done now.